derive/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "zk_evm_derive"
```

Comment:
nit: I'd rather this was called `zk-evm-proc-macro`

Reply:
Happy to call this anything you want, but would underscores be more consistent with the rest of the package names in the workspace?

Also, would you suggest renaming the directory from `derive/` to `proc_macro/` in this case?

Reply:
@gio256 Naming it `zk_evm_proc_macro` to be consistent and renaming `derive` to `proc_macro` folder would be great I think. If we would need another project wide proc macro we would then add it here.